source/includes/gho/_introduction.md

```diff
@@ -63,7 +63,8 @@ on the API Management screen.
 The Greenhouse Onboarding API imposes limits on the amount of data a single client can request over time, as well as the
 complexity of individual requests.  This is done to ensure our servers can always service requests as quickly as possible.
 
-### Request Costs
+### Request Limits
+
```

Comment:
it looks like we have 2 lines between the title and the example request - we should prob only have 1